bin/http2-push-manifest

```diff
@@ -36,6 +36,7 @@ Options:
     -h|--help: print this message
     -v|--version: print version number
     -f|--file: file to discover resources in. Use multiple times to produce a mult-file manifest format.
+    -r|--root: root server path
```

Comment:
can you add an example to the usage? 

Is the intention to fix up paths if your disk structure is different than your server URL structure? One thing I tried to do was [stay out of that business](https://github.com/GoogleChrome/http2-push-manifest/issues/3#issuecomment-154543682) but we can add that if it's convenient.